adagrams/game.py

```diff
@@ -1,13 +1,120 @@
 from random import randint
 
+LETTER_POOL = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B",
+            "C", "C", "D", "D", "D", "D",
+            "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E",
+            "F", "F", "G", "G", "G", "H", "H",
+            "I", "I", "I", "I", "I", "I", "I", "I", "I",
+            "J", "K", "L", "L", "L", "L",
+            "M", "M", "N", "N", "N", "N", "N", "N",
+            "O", "O", "O", "O", "O", "O", "O", "O",
+            "P", "P", "Q",
+            "R", "R", "R", "R", "R", "R",
+            "S", "S", "S", "S",
+            "T", "T", "T", "T", "T", "T",
+            "U", "U", "U", "U", "V", "V",
+            "W", "W", "X", "Y", "Y", "Z"]
+
+SCORE_CHART = {
+        "A" : 1,
+        "E" : 1,
+        "I" : 1,
+        "O" : 1,
+        "U" : 1,
+        "L" : 1,
+        "N" : 1,
+        "R" : 1,
+        "S" : 1,
+        "T" : 1,
+        "D" : 2,
+        "G" : 2,
+        "B" : 3,
+        "C" : 3,
+        "M" : 3,
+        "P" : 3,
+        "F" : 4,
+        "H" : 4,
+        "V" : 4,
+        "W" : 4,
+        "Y" : 4,
+        "K" : 5,
+        "J" : 8,
+        "X" : 8,
+        "Q" :10,
+        "Z" :10
+    }
+
 def draw_letters():
-    pass
+    tile_hand = []
+    LETTER_POOL_copy = LETTER_POOL.copy()
+    while len(tile_hand) < 10:
+            index = randint(0,len(LETTER_POOL_copy) -1)
+            tile_hand.append(LETTER_POOL_copy[index])
+            LETTER_POOL_copy.remove(LETTER_POOL_copy[index])
+
+    return tile_hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+    letter_bank_count = {}
+    
+    for i in letter_bank:
+        if i not in letter_bank_count:
+            letter_bank_count[i] = 1
+        else:
+            letter_bank_count[i] += 1
```

Comment:
Love a frequency map! You could condense this into one line my using the `.get` method available for dictionaries. I would also like to suggest that you name you `i` into something more descriptive such as `letter` or `char`. 
```suggestion
    for letter in letter_bank: 
        letter_bank_count[letter] = letter_bank_count.get(letter, 0) + 1
```